src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() { return false; }
```

Comment:
This is a coding violation. Consider putting your `return` statement to the next line. :)

Reply:
Will take note on that. Thank you.